### DISCUSSION TITLE
ethfs-cli upload 成功之后，查看文件遇到错误 execution reverted: get() must be called on ES node

### DISCUSSION THREAD

**Discussion by linghuccc** [IntensiveCoLearning/Web3-URL#180] [Q&A] [answered]
ethfs-cli create 和 ethfs-cli upload 都运行成功了：
![ethfs](https://github.com/user-attachments/assets/527bd2f3-b603-4034-973b-97c5335a39fd)

当试图查阅文件的时候，遇到下面的错误：
![error](https://github.com/user-attachments/assets/e55bc603-3e65-4abd-b1e6-0d550df96f8f)

URL 如下：
https://0xf31805632f3cf8e2be4759abc61ede19fe295aa4.11155111.w3link.io/welcome.html

**Comment by qzhodl** [comment, marked as answer]
如果在上传数据的时候，使用 `-t 2`，就会将数据上传到ethstorge。这样，在访问的时候 ，就需要使用network id为 3333才行：https://0xf31805632f3cf8e2be4759abc61ede19fe295aa4.3333.w3link.io/welcome.html
